tv2.html

```diff
@@ -0,0 +1,283 @@
+        More from brian tome
+      </h2>
+      <div class="col-sm-6 push-bottom">
+        <crds-portrait-card href="https://www.crossroads.net/media/tags/garage-bible-study/" theme="default"
+          lead="Watch Now" title="garage bible study"
+          image-src="//crds-media.imgix.net/36vvUyQjYYKRzs5gghFNKb/fccad33eb2398fef7160b4092b25f06e/bt-garage.png">
+        </crds-portrait-card>
+      </div>
+      <div class="col-sm-6 push-bottom">
+        <crds-portrait-card href="https://crossroads.net/media/podcasts/the-aggressive-life-with-brian-tome"
+          theme="default" lead="Listen Now" title="Agressive life podcast"
+          image-src="//crds-media.imgix.net/5APtNFVXDhx8uTbHpUc5sD/f2bb379a5b3ad54681f6a3ec37ff7bc5/bt-agressive-life-with-logos.png">
+        </crds-portrait-card>
+      </div>
+    </div>
+    <div class="col-sm-4 d-flex bt-guide-container"
+      style="background-image: url('//crds-media.imgix.net/TB73SSM1V7gnlY8J9PkEC/75c0303bfede033c1508d4cef22e1a2a/bt-map.png'); background-size: cover;">
+      <div class="guide-content">
+        <h2 class="font-family-condensed-extra text-white">YOUR GUIDE TO THE ADVENTUROUS LIFE YOU WERE
+          MADE FOR</h2>
+        <a class="btn btn-white btn-outline fit-content" data-toggle="modal" data-target="#subscribeModalForm"
+          aria-expanded="false" href="#">Subscribe to Brian's Newsletter</a>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="bg-orange soft-ends">
+  <div class="container text-center soft-ends push-ends">
+    <div class="row soft-ends">
+      <h1 class="font-family-condensed-extra text-uppercase text-white">We'd Love to Chat With You</h1>
+      <crds-button color="blue" text="Chat with us" data-trigger="chat"></crds-button>
+    </div>
+  </div>
+</section>
+
+<div aria-labelledby="Subscribe email form" class="modal fade" id="subscribeModalForm" role="dialog" tabindex="-1">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header hard-bottom"><button aria-label="Close" class="close pull-right" data-dismiss="modal"
+          type="button"><span aria-hidden="true">&times;</span></button>
+      </div>
+      <div class="modal-body text-left hard-bottom">
+        <p class="component-header text-center text-gray-dark push-ends soft-bottom" style="line-height:.9em;">Get
+          Words from Brian straight to your inbox.</p>
+        <div class="form" data-role="hsform">
+          <!--[if lte IE 8]>
+                          <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2-legacy.js"></script>
+                          <![endif]-->
+          <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
+        </div>
+      </div>
+      <div class="modal-footer">&nbsp;</div>
+    </div>
+  </div>
+</div>
+
+<style>
```

Comment:
Could this style block be moved to a CSS file? Some declarations seem specific to the page others seem like utility classes. (I tend to not use `<style>` blocks ever unless it is something coming from CMS/JS/Not hardcoded)

Reply:
Agreed. When I am making a page that will end up in the CMS. I always put in the codebase first so if it is absolutely necessary to write more styles for a page then I can put them in the repo. Best thing to do is get as far as you can without writing styles. (we already have so many styles available to us)